grails-app/init/io/xh/toolbox/BootStrap.groovy

```diff
@@ -212,6 +213,40 @@ class BootStrap {
             metricUnit: 'sources',
             active: true
         )
+        createMonitorIfNeeded(
+                code: 'storageSpaceUsed',
+                name: 'Storage Space Used by Uploaded Files',
+                metricType: 'Ceil',
+                metricUnit: 'MiB',
+                warnThreshold: 16,
+                failThreshold: 32,
+                active: true
+        )
+        createMonitorIfNeeded(
+                code: 'maliciousFilesFound',
+                name: 'Malicious Files Uploaded',
+                metricType: 'Ceil',
+                metricUnit: 'potentially malicious files',
+                warnThreshold: 0,
+                failThreshold: 1,
+                active: true
+        )
+        createMonitorIfNeeded(
+                code: 'recallsFetchStatus',
+                name: "Connection status to ${configService.getString('recallsHost', 'api.fda.gov')}",
```

Comment:
Would just say FDA here for the monitor name -- this name is getting fixed in the DB, and this code mistakenly implies that it is dynamic,